README.md

```diff
@@ -2,15 +2,15 @@ Docker Compose
 ==============
 ![Docker Compose](logo.png?raw=true "Docker Compose Logo")
 
-*(Previously known as Fig)*
+Compose is a tool for defining and running multi-container Docker applications.
```

Comment:
Compose is a tool for defining and running multi-container Docker applications.
With Compose, you create in a `docker-compose.yml` configuring your
application's containers. Then, using a single command, you create and start them
all from your configuration. To learn more about all the features of Compose see
[the list of features](#features) on this page.